src/megachatapi_impl.cpp

```diff
@@ -1018,7 +1018,7 @@ void MegaChatApiImpl::sendPendingRequests()
             }
 
             MegaChatCallHandler *handler = findChatCallHandler(chatid);
-            if (handler && handler->getCall())
+            if (handler && !chatroom->isGroup())
```

Comment:
Yes, we don't want to allow. But start request allow start a call and join to a group call. With this code we want to avoid that an user start a call in an 1to1 chatroom where there is a call and  we allow that a user can join in a groupcall

Reply:
Fixed: avoid a second call to be started in the same groupchat